tools/its.py

```diff
@@ -745,26 +769,42 @@ def code_bestaetigen(self, token, sms_pin):
         :return:
         """
 
-        path = f"rest/smspin/verifikation"
+        url = self.plz_to_url(plz_impfzentrum)
+        location = f"{url}rest/smspin/verifikation"
         data = {
             "token": token,
             "smspin": sms_pin
 
         }
+
```

Comment:
```suggestion
cookies = self.get_cookies(url, manual=False)
```
Könnte Issues wie #359 vorbeugen. Denke der Flow ist schöner wenn es erst mal mit `manual=false` versucht. Wenn das nicht klappt mit `manual=true` dann. Neue Cookies werden auf jeden Fall benötigt vill hast du es aber schon so drin und ich übersehe etwas.

Reply:
Ich übergebe per Parameter die cookies vom code anfragen. Das klappt in meinen Tests auch, d. h. an dieser Stelle werden (meistens) keine neuen Cookies benötigt.

Reply:
Stimmt gerade getestet, sehr nice!